### PR TITLE
feat(icon): create Icon component (~Info.LabelIcon)

### PR DESCRIPTION
**Pull request by gijosso**
Rename internal `Icon` to `SvgIcon`
Create a `Icon` component replacing the `<Info><Info.LabelIcon /><Info>`

### CODE REVIEW

File: packages/andive/src/index.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ export {default as DropdownFilter} from './components/dropdown-filter'
 
 export {default as Hover} from './components/hover'
 
-export {default as Icon} from './components/icon'
+export {default as SvgIcon} from './components/svg-icon'
```
Comment (nicobarray) [inline packages/andive/src/index.js]: Don't forget to not export it.

**Review by nicobarray — COMMENTED** [review]
I just checked what was already done (still missing some parts because of its Draft status). We'll merge #42 before this one (it impacts all icon*.js too). Be prepared for _some_ merge conflicts 😬

**Comment by gijosso** [comment]
> Be prepared for _some_ merge conflicts 😬

Yeah @nicobarray was kinda expecting that ahah
I tried to have `Icon` be as customisable as possible (with some flex properties), stories stick to prod's use cases tho

File: packages/andive/src/components/icon.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { ReactNodeLike } from 'prop-types'
+import styled from 'styled-components'
+
+import Box from './box'
+
+const Icon = styled.div<Pick<Props, 'align' | 'direction'>>`
```
Comment (nicobarray) [inline packages/andive/src/components/icon.tsx]: align is a valid HTML attribute, you must filter it like-so:

`const Icon = styled(({ align, direction, ...props }) => <div {...props} />)'   '`

**Review by nicobarray — CHANGES_REQUESTED** [review]
Almost :) Just a StyledComponent to filter.

**Review by nicobarray — APPROVED** [review]
🙏 Thanks

**Pull request merged**